### ISSUE TITLE
Ignore module names

### ISSUE THREAD

**Issue by politician** [aotaduy/eslint-plugin-spellcheck#59]
It should add to the ignore dictionary the modules included in the dependencies as well as native Node.js modules.

### package.json

```
"devDependencies": {
    "rollup": "^2.17.0"
  }
```

### example.js

```
const fs = require('fs')

// Require Rollup
// or import rollup from 'rollup'
const rollup = require('rollup')
```

### spellcheck output

```
  1:7   warning  You have a misspelled word: fs on Identifier      spellcheck/spell-checker
  1:20  warning  You have a misspelled word: fs on String          spellcheck/spell-checker
  3:1   warning  You have a misspelled word: Rollup on Comment     spellcheck/spell-checker
  4:1   warning  You have a misspelled word: rollup on Comment     spellcheck/spell-checker
  4:1   warning  You have a misspelled word: 'rollup' on Comment   spellcheck/spell-checker
  5:7   warning  You have a misspelled word: rollup on Identifier  spellcheck/spell-checker
  5:24  warning  You have a misspelled word: rollup on String      spellcheck/spell-checker
```

Example words added to ignore dictionary for `@scope/module-name`:
- @scope/module-name
- scope
- module-name
- moduleName
- module
- name

**Comment by aotaduy** [comment]
It will be a good idea to automatically add all your node_modules modules to the skipWords, i think this will solve a lot of false positive issue. Im not sure if this is possible from an eslint plugin but i will give it a try

**Comment by aotaduy** [comment]
Maybe you can figureit out and create a PR with sommething like this.

**Comment by Grawl** [comment]
I do it like this:

`.eslintrc.js`

```js
const fs = require("fs");
const path = require("path");

const nodeModules = fs.readdirSync("./node_modules")
  .map(string => string.replace(/^@/, "")); // remove "@" in start of word

const dictionary = fs.readFileSync(path.join(__dirname, "./config/spell.dict"))
  .toString()
  .split('\n')
  .map(word => word.trim())  // Remove any leading/trailing whitespace
  .filter(word => word)  // Remove any empty lines
  .concat(nodeModules);

module.exports = {
  ...
  rules: {
    "spellcheck/spell-checker": ["warn", {
      "minLength": 5,
      skipWords: dictionary,
      skipIfMatch: [
        "https?://[^s]*",
        "^[-\\w]+\/[-\\w\\.]+$", //For MIME Types
        // "^@.*" // for mentions like @name
      ]
    }]
  },
}
```

original idea from this comment: https://github.com/aotaduy/eslint-plugin-spellcheck/issues/32#issuecomment-537724475

**Comment by aotaduy** [comment]
I think thius is a good solution and does not requre extra coding on the plugin thank you very much !
